BuilderPattern.java

```diff
@@ -0,0 +1,54 @@
+package Ex4;
+public class BuilderPattern{
+	public static void main(String[] args) {
+        Car myCar=new Car.BuildCar().setCompany("Honda").setModel("Civic").setEngine("Petrol").setTire("MRF").build();
+        System.out.println(myCar);
+	}
+}
```

Comment:
Always have some gap between main class and other class

Reply:
Ok Bro